dstest/dstest.go

```diff
@@ -94,6 +96,15 @@ func (t TestCase) VizScriptFile() (qfs.File, bool) {
 	return qfs.NewMemfileBytes(t.VizScriptFilename, t.VizScript), true
 }
 
+// RenderedFile returns a qfs.File of the rendered file if one exists
+func (t TestCase) RenderedFile() (qfs.File, error) {
+	path := filepath.Join(t.Path, RenderedFilename)
+	if f, err := os.Open(path); err == nil {
+		return qfs.NewMemfileReader(RenderedFilename, f), nil
+	}
+	return nil, os.ErrNotExist
```

Comment:
Should return err here instead of os.ErrNotExist. There could be other causes that `Open` fails, like a permissions problem, or a network timeout (NFS filesystem), or a harddrive that spontaneously caught on fire.